### DISCUSSION TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'img/minisectors_lap_46.png'

### DISCUSSION THREAD

**Discussion by tejasgowdaki** [theOehrly/Fast-F1#191] [Q&A] [answered]
I'm trying to get fastest lap results and getting this error for some reason. And it's happening for all the queries i'm doing. Tried disabling the cache and clearing it, nothing is working. Please help me with this.

<img width="1418" alt="Screenshot 2022-06-12 at 12 25 49 AM" src="https://user-images.githubusercontent.com/23277573/173201403-d1537815-340f-417f-8223-5b42dfa1109e.png">

**Comment by theOehrly** [comment, marked as answer]
I'm almost certain that this is not directly related to FastF1. It's failing to save an image file and no part of FastF1 does that.
Cannot really conclude more without seeing any code. 
Most likely your missing a folder 'img' in your working directory, though.
